human:
  group_name: "human_right_arm"
  recipe:
    A1: "blue_box"
    A3: "white_box"



robot:
  group_name: "ur5_on_guide"
  recipe:
    A4: "orange_box"
    A2: "blue_box"


